app/views/providers/capital_introductions/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= page_template page_title: t('.h1-heading') do %>
+  <%= simple_format t('.text_1') %>
```

Comment:
I think this would be more readable if the text was labelled more meaningfully (see translation file)